docs/DEV.md

```diff
@@ -37,14 +37,6 @@ $ npm run build
 
 This packages the add-on into an xpi file which is stored in `dist/`. This file is what you load into Firefox.
 
-## General notes on Shield Study Engineering
```

Comment:
I think this block provides some important context for devs on embedded webextensions; why did you remove it?

Reply:
I found there was two duplicate sections about this, so I removed one of them.